.github/workflows/test.yml

```diff
@@ -34,12 +33,9 @@ jobs:
         exclude:
         - { django-version: '3.2', python-version: '3.11' }
         - { django-version: '3.2', python-version: '3.12' }
-        - { django-version: '4.1', python-version: '3.6' }
         - { django-version: '4.1', python-version: '3.7' }
         - { django-version: '4.1', python-version: '3.12' }
-        - { django-version: '4.2', python-version: '3.6' }
         - { django-version: '4.2', python-version: '3.7' }
```

Comment:
Might as well get rid of this too: https://endoflife.date/python as of june of last year.

Reply:
[Django 3.2 LTS](https://pypi.org/project/Django/3.2.24/) supports Python 3.6 through 3.10 according to the metadata on PyPI. It's more for that that it makes sense to keep Python 3.7 around.

But we can certainly drop the obsolete Django 4.1. See #157